sammba/registration/struct.py

```diff
@@ -11,7 +11,8 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
                     nonlinear_levels=[1, 2, 3],
                     nonlinear_minimal_patches=[75],
                     nonlinear_weight_file=None,
-                    convergence=0.005, caching=False, verbose=True,
+                    convergence=0.005, two_blur=1.1,
```

Comment:
`two_blur` is not very informative, what about `blur_radius`

Reply:
please also add this parameter in doc

Reply:
done